doc/cli/npm-shrinkwrap.md

```diff
@@ -158,12 +158,13 @@ wouldn't actually work. Similarly, the command will fail if there are
 extraneous packages (not referenced by `package.json`), since that would
 indicate that `package.json` is not correct.
 
-Since `npm shrinkwrap` is intended to lock down your dependencies for
-production use, `devDependencies` will not be included unless you
-explicitly set the `--dev` flag when you run `npm shrinkwrap`.  If
-installed `devDependencies` are excluded, then npm will print a
-warning.  If you want them to be installed with your module by
-default, please consider adding them to `dependencies` instead.
+Starting with npm v4.0.0, `devDependencies` are included when you run
+`npm shrinkwrap` and follow the usual rules as to when they're installed.
+That is, if you run `npm install --production` or
+`npm install --only=production` they won't be installed. Similarly, if
+the environment variable `NODE_ENV` is `production` then they won't be
+installed. You can request they not be included with:
+`npm shrinkwrap --production` or `npm shrinkwrap --only=prod`.
```

Comment:
What do you think about flipping the order of these around. It would be nice to (eventually) deprecate `--prod` and `--dev` and only use the clearer `--only` and `--also` in their place.